Releases.md

```diff
@@ -34,6 +35,24 @@ Semantic versioning is used since the version number is able to convey clear inf
 
   A major release will also likely require a change of the container manager version used, for example Docker\*. Please refer to the release notes for further details.
 
+## Components
+
+A new release will result in all Kata components being given a new [version](#versioning), even if no changes were made to that component since the last version. The version for a release is **identical** for all  components.
```

Comment:
Suggested rewrite: 
A new release results in all Kata components being given a new [version](#versioning), even if there are no changes to that component since the last version.